src/main/java/ui/UI.java

```diff
@@ -522,4 +526,20 @@ public void updateTitle() {
         mainStage.setTitle(title);
     }
     
+    public String getTitle() {
+        return mainStage.getTitle();
+    }
+    
+    public int getVersionMajor() {
+        return VERSION_MAJOR;
```

Comment:
Do we really need these getters? They're constants, so we can just make them public I guess.